static/src/javascripts/projects/commercial/modules/third-party-tags/outbrain.js

```diff
@@ -114,6 +117,37 @@ define([
             detect.adblockInUseSync();
     }
 
+
+    function checkDependencies() {
+        return Promise.all([checkEmailSignup(), checkClashingABTest()])
+            .then(function(result) {
+
+                function findEmail(value) {
+                    return value == 'email';
+                }
+
+                return result.find(findEmail);
+            })
+            .catch(function () {
+                return 'email';
+            });
+    }
+
+    function checkClashingABTest() {
+        if (!clashingABTestPromise) {
+            clashingABTestPromise = new Promise(function (resolve) {
+                if (clash.userIsInAClashingAbTest()) {
+                    resolve('email');
```

Comment:
Could we possibly update this to a more generic `non-compliant` as it's not specific to email now?